src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -47,18 +46,14 @@ public RetrieveController(IMediator mediator, ILogger<RetrieveController> logger
             _logger = logger;
         }
 
-        [AcceptContentFilter(new[] { KnownContentTypes.ApplicationDicom }, allowSingle: false, allowMultiple: true)]
```

Comment:
Are these filters used anywhere else? If not should we delete them?

Reply:
AcceptContentFilter is still used by GetxxMetadataAsync, since GetMetadata has a different code path, I didn't plan to touch them.